docs/zcml.rst

```diff
@@ -15,15 +15,15 @@ This helper will let us easily execute ZCML snippets:
 
 .. doctest::
 
-   >>> from cStringIO import StringIO
+   >>> from io import BytesIO
    >>> from zope.configuration.xmlconfig import xmlconfig
    >>> def runSnippet(snippet):
    ...     template = """\
    ...     <configure xmlns='http://namespaces.zope.org/zope'
    ...                i18n_domain="zope">
    ...     %s
    ...     </configure>"""
```

Comment:
No, native strings. They then get encoded into bytes later. (This was simpler than using `io.StringIO` and adding `u` prefixes everywhere for Python 2.)